Oleksii_Fedorchuk/HW8_Oleksii_Fedorchuk/task2.py

```diff
@@ -0,0 +1,23 @@
+from typing import Callable
+
+if __name__ == "__main__":
+    list_of_items = ["BMW", "Bugatti", "Mercedes", "Volkswagen"]
+
+
+    def filter_func(func: Callable, item: list | tuple) -> list:
+        filter_list = []
+        for items in item:
+            if func(items):
+                filter_list.append(items)
+        return filter_list
+
+
+    def checking(item: list) -> bool:
```

Comment:
not necessary to define this function. You can use lambda in your function as well